python/pyspark/sql/tests/connect/test_connect_function.py

```diff
@@ -2210,15 +2210,39 @@ def test_call_udf(self):
             ).toPandas(),
         )
 
+    def test_udf(self):
+        from pyspark.sql import functions as SF
+        from pyspark.sql.connect import functions as CF
+
+        query = """
+            SELECT a, b, c, BINARY(c) as d FROM VALUES
+            (-1.0, float("NAN"), 'x'), (-2.1, NULL, 'y'), (1, 2.1, 'z'), (0, 0.5, NULL)
+            AS tab(a, b, c)
+            """
+
+        # +----+----+----+----+
+        # |   a|   b|   c|   d|
+        # +----+----+----+----+
+        # |-1.0| NaN|   x|[78]|
+        # |-2.1|null|   y|[79]|
+        # | 1.0| 2.1|   z|[7A]|
+        # | 0.0| 0.5|null|null|
+        # +----+----+----+----+
+
+        cdf = self.connect.sql(query)
+        sdf = self.spark.sql(query)
+
+        self.assert_eq(
+            cdf.select(CF.udf(lambda x: x + 1)(cdf.a)).toPandas(),
+            sdf.select(SF.udf(lambda x: x + 1)(sdf.a)).toPandas(),
```

Comment:
on `pyspark.sql.function.udf` we need a similar `@try_remote_functions`. This will help in the support.

Reply:
Added.